release-notes/docs/3.9.0.md

```diff
@@ -0,0 +1,163 @@
+
+### Supported software
+
+There are no required changes to the [supported software matrix](https://docs.communityhealthtoolkit.org/core/overview/supported-software/#dependencies) from `3.0.0`.
+
+| Node | CouchDB | Browsers | SMS bridge | Android | medic-android | medic-couch2pg |
+|----|----|----|----|----|----|---|
+| 8.11+ | 2.1+ | Chrome 53+, Firefox latest | medic-gateway | 4.4+ | 0.4.5+ | 3.0+ |
+
+## DHIS2 integration
+
+The CHT now supports calculating and exporting data for [DHIS2](https://www.dhis2.org) integration. The Core Framework can now be configured to calculate DHIS2 dataValues, aggregate patient level data, view aggregate data offline, and export the aggregate data formatted for DHIS2. The data is also available via an API so you can access it from an interoperability layer such as OpenHIM.
+
+For more information read [the documentation](https://docs.communityhealthtoolkit.org/apps/features/integrations/dhis2/).
+
+Issues: [cht-core#5661](https://github.com/medic/cht-core/issues/5661) and [cht-core#6002](https://github.com/medic/cht-core/issues/6002)
+
+## Supervise target progress
+
+Targets can now be configured to be aggregated and synced to allow supervisors to view the progress of their health workers towards their goals. These new "aggregate" targets show an overview of all workers and can be selected to show each worker's progress individually.
+
+Read [the documentation](https://docs.communityhealthtoolkit.org/apps/reference/targets/#targetsjs) to see how to configure this new aggregated target.
+
+Issues: [cht-core#4839](https://github.com/medic/cht-core/issues/4839) and [cht-core#4840](https://github.com/medic/cht-core/issues/4840)
+
+## RapidPro integration
+
+The Outbound Push feature has been extended to support submitting data to RapidPro so you can now trigger workflows based on changes in the CouchDB database.
+
+Read more on our [Outbound Push documentation page](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/outbound/).
+
+Issues: [cht-core#6207](https://github.com/medic/cht-core/issues/6207)
+
+## Case management workflows
+
+For reports that aren't patient or place centric you can now register a case with a generated `case_id` short code. This allows subsequent reports to identify which case they refer to.
+
+Read more about configuring the `accept_case_reports` transition in the [CHT documentation](https://docs.communityhealthtoolkit.org/apps/reference/app-settings/transitions/).
+
+Issues: [cht-core#6291](https://github.com/medic/cht-core/issues/6291).
+
+## Instance monitoring
+
+A [new API](https://github.com/medic/cht-core/blob/master/api/README.md#monitoring) has been added to allow tracking of a range of metrics in the CHT platform to make monitoring, alerting, and debugging of your instance easier.
```

Comment:
I think @abbyad is in the process of moving api documentation to the doc site. It's not live yet, but link should be changed before publishing.

Reply:
Pending merge: https://github.com/medic/cht-docs/pull/142

Reply:
Updated